src/main/java/com/snack/news/controller/NewsController.java

```diff
@@ -1,13 +1,12 @@
 package com.snack.news.controller;
 
 import com.snack.news.domain.news.News;
-import com.snack.news.dto.NewsDto;
-import com.snack.news.dto.WrappedResponse;
+import com.snack.news.dto.*;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unused imports such as 'com.snack.news.dto'](https://app.codacy.com/manual/snack-news/Snack-BE/pullRequest?prid=4962836)